### DISCUSSION TITLE
Error [40] help: QGIS

### DISCUSSION THREAD

**Discussion by B2Nic** [semiautomaticgit/SemiAutomaticClassificationPlugin#281] [Q&A]
Hi, I am using the plugin with QGIS 3.28.5 and up until about 2 months ago I had no problems vi3ewing or downloading data. Now I seem to be getting _'_Error [40]: Unable to connect, possibly archived image'__ consistently for all of the sensors, regardless of the date I choose. Not all sensors or images but the vast majority of them. I have reinstalled both QGIS and the plugin. Any help is appreciated. Thank you.

**Comment by semiautomaticgit** [comment]
Hi @B2Nic , unfortunately there are a few issue with the services at the moment. 
In October the new version 8 of the plugin should solve these issues.
